frontend/src/component/ContentListItem.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import { Link } from 'react-router-dom'
+
+import {
+  Breadcrumbs,
+  CONTENT_TYPE,
+  Icon,
+  ListItemWrapper,
+  FilenameWithExtension,
+  PAGE
+} from 'tracim_frontend_lib'
+import ContentType from './ContentType.jsx'
+import TimedEvent from './TimedEvent.jsx'
+
+require('./ContentListItem.styl')
+
+const getRevisionTypeLabel = (revisionType, t) => {
+  switch (revisionType) {
+    case 'revision':
+      return t('modified')
+    case 'creation':
+      return t('created')
+    case 'edition':
+      return t('modified')
+    case 'undeletion':
+      return t('undeleted')
+    case 'mention':
+      return t('mention made')
+    case 'content-comment':
+      return t('commented')
+    case 'status-update':
+      return t('status modified')
+    case 'unknown':
+      return t('unknown')
+  }
+
+  return revisionType
+}
+
+const numberCommentsTitle = (numberComments, t) => {
+  if (numberComments === 0) return t('0 comments')
+  if (numberComments === 1) return t('{{numberComments}} comment', { numberComments: numberComments })
```

Comment:
Let's simplify?

```suggestion
  if (numberComments === 1) return t('1 comment')
```

Reply:
It would simplify the code, but not the translation file…

Reply:
even if the other string is '{{numberComments}} comments' (with a s)?

Reply:
OK, I assumed it used the `_plural` functionality of i18next, but it seems that this functionality is not well-known…